lib/Common/Memory/PageAllocator.cpp

```diff
@@ -299,6 +301,11 @@ PageSegmentBase<T>::IsAllocationPageAligned(__in char* address, size_t pageCount
         return true;
     }
 
+    if (nextIndex != nullptr)
+    {
+        *nextIndex = (uint) ((reinterpret_cast<uintptr_t>(address) % (mask + 1)) / AutoSystemInfo::PageSize);
```

Comment:
Does nextIndex need to be computed this way or would it also work to just set nextIndex to pageCount? (in which case we don't need this out parameter in the first place)